fdbclient/vexillographer/fdb.options

```diff
@@ -200,7 +203,10 @@ description is not currently required but encouraged.
     <Option name="max_retry_delay" code="502"
             paramType="Int" paramDescription="value in milliseconds of maximum delay"
             description="Set the maximum amount of backoff delay incurred in the call to ``onError`` if the error is retryable. Defaults to 1000 ms. Valid parameter values are ``[0, INT_MAX]``. If the maximum retry delay is less than the current retry delay of the transaction, then the current retry delay will be clamped to the maximum retry delay. Prior to API version 610, like all other transaction options, the maximum retry delay must be reset after a call to ``onError``. If the API version is 610 or greater, the retry limit is not reset after an ``onError`` call. Note that at all API versions, it is safe and legal to set the maximum retry delay each time the transaction begins, so most code written assuming the older behavior can be upgraded to the newer behavior without requiring any modification, and the caller is not required to implement special logic in retry loops to only conditionally set this option."/>
-    <Option name="snapshot_ryw_enable" code="600"
+    <Option name="max_transaction_size" code="503"
```

Comment:
Should this be `size_limit`? I think it's fine if the variables inside are `max_transaction_size`, but I think we want this to be `size_limit` so that the transaction option is just `setSizeLimit` and then the database option is `setTransactionSizeLimit` (which has parallelism with the other database/transaction option pairs).